solver/src/solver/http_solver/model.rs

```diff
@@ -56,21 +59,23 @@ pub struct TokenInfoModel {
 
 #[derive(Debug, Serialize)]
 pub struct CostModel {
+    #[serde(with = "serde_with::rust::display_fromstr")]
     pub amount: u128,
-    pub token: String,
+    pub token: H160,
 }
 
 #[derive(Debug, Serialize)]
 pub struct FeeModel {
+    #[serde(with = "serde_with::rust::display_fromstr")]
     pub amount: u128,
-    pub token: String,
+    pub token: H160,
 }
 
 #[derive(Debug, Deserialize)]
 pub struct SettledBatchAuctionModel {
     pub orders: HashMap<String, ExecutedOrderModel>,
     pub uniswaps: HashMap<String, UpdatedUniswapModel>,
```

Comment:
```suggestion
    pub orders: HashMap<usize, ExecutedOrderModel>,
    pub uniswaps: HashMap<usize, UpdatedUniswapModel>,
```

Reply:
This was left as is because I assumed incoming data should be left as String, but I will change it.